jinja_utils.py

```diff
@@ -75,12 +105,12 @@ def parse_string(string, params, autoescape=True):
     """Parses a string using Jinja templating.
 
     Args:
-      string: the string to be parsed.
-      params: the parameters to parse the string with.
-      autoescape: whether to enable autoescaping when parsing.
+        string: str. The string to be parsed.
+        params: list(*). The parameters to parse the string with.
```

Comment:
How do you know it's a list? (Make sure to run it in context; do not guess.)

Reply:
Please see the above comment.